premailer/premailer.py

```diff
@@ -409,6 +411,18 @@ def transform(self, pretty_print=True, **kwargs):
                 parent = item.getparent()
                 del parent.attrib['class']
 
+        # Add align attributes to images if they have a CSS float value of
+        # right or left. Outlook (both on desktop and the web version) are
+        # bad at parsing CSS (like floats) but they do understand the HTML
+        # align attribute.
+        if self.align_floating_images:
+            for item in page.xpath("//img"):
```

Comment:
single quotes please. 
I know it doesn't make any run-time difference but good code tries to be consistent.